library/core/src/ptr/mod.rs

```diff
@@ -1115,12 +1080,12 @@ pub unsafe fn read_volatile<T>(src: *const T) -> T {
 ///
 /// Note that even if `T` has size `0`, the pointer must be non-NULL and properly aligned.
 ///
-/// [valid]: ../ptr/index.html#safety
+/// [valid]: #safety
 ///
 /// Just like in C, whether an operation is volatile has no bearing whatsoever
 /// on questions involving concurrent access from multiple threads. Volatile
 /// accesses behave exactly like non-atomic accesses in that regard. In particular,
-/// a race between a `write_volatile` and any other operation (reading or writing)
+/// a race between a [`write_volatile`] and any other operation (reading or writing)
```

Comment:
This is the current function, is it worth linking to?

Reply:
Good point, I guess it doesn't offer much :) And it might be a bit confusing.